src/OrchardCore/OrchardCore.Navigation.Core/NavigationHelper.cs

```diff
@@ -11,6 +10,9 @@ namespace OrchardCore.Navigation
 {
     public class NavigationHelper
     {
+        public static bool IsAdminMenu(string name)
+            => NavigationConstants.AdminMenuName == name;
```

Comment:
I'd do that but we consciously don't use expression-bodied methods.

```suggestion
        public static bool IsAdminMenu(string name)
        {
            return NavigationConstants.AdminMenuName == name;
        }
```

Reply:
We use expression bodied methods all the time in OC. I see no need for this. This method isn't expected to change.

Reply:
Did the recommendation change? OK then.